src/services/data-service.js

```diff
@@ -1,11 +1,41 @@
 import { Post } from '../models/post';
 
-import firebaseReference from "../firebaseConfig"
+import firebaseReference from "../firebaseConfig";
 import 'firebase/firestore';
 
 export class DataService {
-	constructor() {
-	}
+	constructor() {}
+
+  getPromptsFromDB(state) {
+      firebaseReference.collection('days')
+      .limit(1) //only get the first prompt from the database
+      .get()
+        .then(snapshot => {
+          snapshot.forEach(doc => {
+            if (!doc.exists) {
+              console.log('No such document!');
+            } else {
+              this.displayPromptsFromDB(doc.data().prompt, state)
+            }
+          })
+        }) 
+          .catch(err => {
+            console.log('Error getting documents', err);
+          });
+
+  }
+
+  //sets the prompt's state to the same one as DB
+  displayPromptsFromDB(prompt, state) {
+    state.setState({
+      title: prompt
+    });
+  }
+
+  updatePromptsFromDB(state, state_title) {
+    firebaseReference.collection('days').doc(state_title).delete(); //deletes the old state
```

Comment:
Does this delete the previous prompt from the db?

Reply:
yes!

Reply:
Do we want it to if we have an archives section?

Reply:
I also don't think client-side code should be responsible for deleting prompts.

Reply:
hmmm interesting, discuss at meeting?